### ISSUE TITLE
How often ( if ever ) quantity value is synced? 

### ISSUE THREAD

**Issue by serpico** [openfoodfacts/open-prices#315]
Epic: #

### Description
I noticed the case with [3564700009321](https://prices.openfoodfacts.org/app/products/3564700009321) for which monsieurtanuki kindly corrected the quantity value 7 days ago on OFF ( it seems to be kind of a special case since the data come from the manufacturer and are not modifiable straight away ( from his explanations app users have to activate the dev mode - for website users I don't know...).

I'm also wondering about values like categories, which could be useful to compare price for similar products although not identical by going deeper into the category tree ( comparing price per quantity for dried red beans only instead of comparing both canned and dried  red beans ).

edit: also bought today ` 4032489010108` quantity updated 2024-06-08 to 500g on OFF as mentioned on the packaging ( though I didn't update the picture of said packaging ) just to know if and when the weight will be synced on OP

**Comment by raphodn** [comment]
Good question, the products should be synced every day..

In the OFF API : https://world.openfoodfacts.org/api/v2/product/3564700009321.json
- I see `"last_modified_t":1716977371` which corresponds to `Wednesday, May 29, 2024 12:09:31 PM GMT+02:00 DST`
- and `"product_quantity":"225","product_quantity_unit":"g","quantity":"225g","quantity_imported":"2250 g"`

In the OP API : https://prices.openfoodfacts.org/api/v1/products/code/3564700009321
- I see `"updated": "2024-05-30T10:31:49.531738Z"`
- but `"product_quantity": 2250,`

**Comment by raphael0202** [comment]
I found the culprit: the dataset generation ends after 10AM (~12PM), and the job is scheduled at 10AM.
So we assume that the downloaded dump is the dump of the day, whereas it's the dump of the previous day.
I updated the `source_last_synced` field to the 27th of May and relaunched manually the import to make sure that the error comes from the old dump download.

If confirmed, we should delay the import (~3PM for example) and relaunch a full import.
